app/src/components/modal/modal_your_connection/index.tsx

```diff
@@ -288,7 +289,7 @@ export const ModalYourConnection = (props: Props) => {
               <TopCardHeaderLeft>
                 <ConnectionIconWrapper>
                   <ConnectorCircle>{connectorIcon}</ConnectorCircle>
-                  <IconJazz account={account || ''} size={28} />
+                  <IconJazz account={context.rawWeb3Context.account || ''} size={28} />
```

Comment:
can`t we just use the owner variable which has already be assigned to `context.rawWeb3Context.account` ?

Reply:
Looks like the owner variable is used in the function scope, but would be good to assign it at the component level once and use it everywhere that way

Reply:
i tried using that var but got error. i will check again